Services/UserService/Commands/EditUserCommand.cs

```diff
@@ -29,11 +37,52 @@ public Task<bool> Execute(EditUserRequest request)
                     || string.IsNullOrEmpty(passwordRequest.NewPassword)))
             {
                 passwordChangeValidator.ValidateAndThrow(passwordRequest);
+            } else
+            {
+                throw new BadRequestException("Password is null or empty");
             }
             userInfoValidator.ValidateAndThrow(userInfo);
 
-            // TODO Realise edit user logic
-            throw new NotImplementedException();
+            string oldPasswordHash = ShaHash.GetPasswordHash(request.PasswordRequest.OldPassword);
+            string newPasswordHash = ShaHash.GetPasswordHash(request.PasswordRequest.NewPassword);
+            var user = new User
```

Comment:
Email also, I think . But it depens on what exactly we will have form GUI

Reply:
Fixed